cloud_snitch/migrate/uuid.py

```diff
@@ -0,0 +1,321 @@
+import time
+
+from cloud_snitch.cli_common import base_parser
+from cloud_snitch.cli_uuid import get_neo4j
+from cloud_snitch.models import registry
+from cloud_snitch.utils import EOT
+
+from neo4j.exceptions import DatabaseError
+
+
+class Property:
+
+    def __init__(self, label, prop):
+        self.label = label
+        self.property = prop
+
+    def __str__(self):
+        return '{}.{}'.format(self.label.lower(), self.property)
+
+
+class IdentityChangeQuery:
+    """Models a query to change the identity of nodes in a subgraph."""
+
+    delimiter = '_'
+
+    def __init__(self, label, identity_properties, path):
+        """Init the query.
+
+        :param label: Name of the label
+        :type label: str
+        :param identity_properties: Collection of properties
+        :type identity_properties: tuple|list
+        :param path: Path to the label(usually from the registry)
+        :type path: List of label, reltype tuples
+        """
+        self.label = label
+        self.identity_properties = identity_properties
+        self.path = path
+
+    def match_clause(self):
+        """Builds the match portion of the query.
+
+        :returns: Match clause of a statement
+        :rtype: str
+        """
+        cipher = 'MATCH '
+        for label, reltype in self.path:
+            cipher += '({}:{})-[:{}]->'.format(
+                self.var_from_label(label), label, reltype
+            )
+        cipher += '({}:{})'.format(self.var_from_label(self.label), self.label)
+        return cipher
+
+    def set_clause(self):
+        """Builds the set clause of the query.
+
+        :returns: Set clause of the query.
+        :rtype: str
+        """
+        value = []
+        for prop in self.identity_properties:
+            value.append(str(prop))
+        value = ' + \'_\' + '.join(value)
+        cipher = 'SET {}.{} = {}'.format(
+            self.label.lower(),
```

Comment:
shouldn't this be `self.var_from_label(self.label)` for consistency?

Reply:
Good catch.